catalogue/mpi/pdq/user-guide/overview.md

```diff
@@ -0,0 +1,104 @@
+## System Overview
+
+The Patient Demographics Query Service (PDQ) enables application suppliers to query the NHS Wales Master Patient Index (MPI) by specific search criteria for a list of patients comprising patient demographic and identifier details. 
+
+The service is based on the [PDQ-ITI-21](https://oehf.github.io/ipf-docs/docs/ihe/iti21/) transaction from the Integrating the Healthcare Enterprise (IHE) [Patient Demographics Query](https://wiki.ihe.net/index.php/Patient_Demographics_Query) profile. Consumers of the service query a Master Patient Index which presents a consolidated view (the ‘Composite View’) of patient demographic and identifier data assembled from Local Health Board GP, Patient Administration and Departmental Systems implemented in Wales. 
+
+The MPI is by design an [enterprise master patient index](https://en.wikipedia.org/wiki/Enterprise_master_patient_index "Enterprise master patient index") whose aim is to correctly identify patients whenever or wherever they receive treatment. The result of linking patient identity records across a range of information systems is a single ‘gold standard’ identity record for use by new national systems. This helps minimise the number of duplicate records and supports health board system mergers.
+
+### Message Format 
+This Service exposes an HTTP SOAP web service endpoint.
+
+The NHS Wales PDQ (Query Parameter Definition) messages conform to the Integrating the Healthcare Enterprise (IHE) Patient Demographics Query (PDQ) profile [HL7v2.5](https://hl7-definition.caristix.com/v2/HL7v2.5/Segments). 
+
+These consist of a QBP^Q22^QBP^Q21 request and corresponding synchronous RSP^K21 responses as per standard HL7.
```

Comment:
It's technically a `QPB_Q21` Message Structure triggered by a `Q22` event (of type `QBP`) but as discussed, maybe not something for the overview and better places in a "concepts" section.

Reply:
I can drop line 14 out for now if that would be better. And we could add a task to include a concepts section at a later date. That bit was from pg7 of the document in sharepoint.  
https://nhswales365.sharepoint.com/:w:/r/sites/DigitalHealthEcosystem/_layouts/15/Doc.aspx?sourcedoc=%7B178FD27B-CB8D-4955-A89B-C93C6E3B0AF2%7D&file=Patient%20Demographics%20Query%20Service%20-%20External%20Interface%20Specification%20v1.0.doc&action=default&mobileredirect=true

Reply:
Conclusion I think I'll leave that line in there for now. We can always revisit should we add a "concepts" section at a later point.